lms/djangoapps/student_account/views.py

```diff
@@ -567,6 +567,22 @@ def account_settings_context(request):
         'order_history': user_orders
     }
 
+    enterprise_customer_name = None
+    sync_learner_profile_data = False
+    enterprise_learner_data = get_enterprise_learner_data(site=request.site, user=request.user)
+    if enterprise_learner_data and 'enterprise_customer' in enterprise_learner_data[0]:
```

Comment:
Is that possible a Learner is not associated with enterprise customer in fetch `results`? 
why is it necessary to check ---- `'enterprise_customer' in enterprise_learner_data[0]` ?

Reply:
I am not sure if it is possible for a Learner which is present in enterprise API, to not link with enterprise customer. It seems we are not checking this condition on other places so I am going to remove it.